src/utils/balance-utils.js

```diff
@@ -1,10 +1,23 @@
 export function getTotalUnlockedActiveBalance({
-  activeBalance,
   lockedBalance,
+  activeBalance,
 }) {
   return activeBalance.amount.sub(lockedBalance.amount)
 }
 
+export function getTotalLockedANJDistribution({
+  lockedBalance,
+  inactiveBalance,
+}) {
+  if (!lockedBalance.distribution && inactiveBalance.amountNotEffective.eq(0))
```

Comment:
Small thing, but let's prefer braces :).

Perhaps we can change the eslint rule to reflect this.